src/js/tabs/onboard_logging.js

```diff
@@ -156,7 +156,16 @@ TABS.onboard_logging.initialize = function (callback) {
                          analytics.sendEvent(analytics.EVENT_CATEGORIES.FLIGHT_CONTROLLER, 'RebootMsc');
 
                         var buffer = [];
-                        buffer.push(2);
+                        if (semver.gte(CONFIG.apiVersion, "1.41.0")) {
+                            if (GUI.operating_system === "Linux") {
+                                buffer.push(3);  // Reboot into MSC using UTC time offset instead of user timezone
```

Comment:
I know it's my fault for not introducing constants here in the first place, but I think now that there's more than one value, it's definitely time to introduce them.

Reply:
Ok, I can extend the types already defined in MspHelper like:
```
    self.REBOOT_TYPES = {
        FIRMWARE: 0,
        BOOTLOADER: 1,
        MSC: 2,
        MSC_UTC: 3
    };
```
But I don't know how to reference this outside of MspHelper (or if that's possible) for example from onboard_logging.js. Inside of MspHelper it's referenced like `self.REBOOT_TYPES.MSC`

(I don't work with Javascript much...)

Reply:
You can reference them as a property on the global `mspHelper` variable that is exposed in `serial_backend.js`: `mspHelper.REBOOT_TYPES`.

Reply:
Thanks. Got it working.

Extended and changed the logic to use the constants instead of literal reboot type values. Also updated the call from setup.js to enter DFU.